vars/operatorPipeline.groovy

```diff
@@ -13,9 +13,12 @@ def call(Map pipelineParams) {
         environment {
             AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
             AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
-            SCT_CLUSTER_BACKEND   = 'k8s-gce-minikube'
+            SCT_CLUSTER_BACKEND   = "${pipelineParams.get('backend', params.backend)}"
```

Comment:
pipelineParams is readonly map, .get in  groovy adds value that is requested to the map, so it will fail if backend is not defined.
Following construction solves the problem:
"${tmp ? tmp : 'aws'}"

Reply:
This thing doesn't affect `pipelineParams`. This is a problem with `params`.

But thank you for sharing the knowledge about `.get()` :)